### ISSUE TITLE
Line length should not be longer than 75 characters

### ISSUE THREAD

**Issue by wadouk** [adamgibbons/ics#65]
Nothing seems to be done to limit the line length, especially for description field
When i try to validate the result I have this error, so any ical clients could refuse the generated calendar

Line length should not be longer than 75 characters near line #11
Reference: RFC 5545 3.1. Content Lines
http://icalendar.org/iCalendar-RFC-5545/3-1-content-lines.html

**Comment by ryanwitt** [comment]
I recall running into problems with outlook for .ics files that don't follow this spec.

Found the following code in my custom .ics generator, where `@line` needs to be the entire line, including the `LABEL:`

```javascript
  function foldLine (line) {
    const parts = []
    while (line.length > 75) {
      parts.push(line.slice(0, 75))
      line = line.slice(75)
    }
    parts.push(line)
    return parts.join('\r\n\t')
  }
```

I'm considering a PR, but am looking into other changes needed for my use case first.

**Comment by ryanwitt** [comment]
Meanwhile @wadouk, I *think* you could use the above function on the input if you added, then subtracted the label first, i.e.

```javascript
const event = {
  ...
  description: foldLine('DESCRIPTION: ' + longLine).slice('DESCRIPTION: '.length),
  ...
}
```

**Comment by adamgibbons** [comment]
Closed by #66 (thanks again @ryanwitt).
